main.cpp

```diff
@@ -498,6 +641,26 @@ struct Airplane
     void dispatchToAirTraffic();
 };
 
+void Airplane::fly()
+{
+    std::cout << "Airplane taking off." << std::endl;
+}
+void Airplane::transportPassengers(int passengerAmount)
+{
+    if(passengerAmount <= fuselage.capacity)
+    {
+    std::cout << "Transporting " << passengerAmount << " passengers." << std::endl;
+    }
+    else
+    {
+    std::cout << "Cannot transport " << passengerAmount << " passengers. Maximum capacity is " << fuselage.capacity << "." << std::endl;
+    }
+}
```

Comment:
Code inside of Curly braces needs to be indented for spaces relative to those Curly braces
- [ ] revise